src/GAP/iso_oscar_gap.jl

```diff
@@ -211,6 +209,50 @@ end
 #TODO function iso_oscar_gap(F::T) where T <: QabField
 
 
+################################################################################
+#
+# Univariate polynomial rings
+#
+@attributes FmpqPolyRing # TODO: port this to Nemo
+@attributes FmpzPolyRing # TODO: port this to Nemo
+@attributes FqNmodPolyRing # TODO: port this to Nemo
+@attributes FqPolyRing # TODO: port this to Nemo
+@attributes GFPFmpzPolyRing # TODO: port this to Nemo
+@attributes GFPPolyRing # TODO: port this to Nemo
+#TODO: support
+# `NmodPolyRing` (from `Nemo.NmodRing(UInt64(6))`)
+# `FmpzModPolyRing` (from `Nemo.FmpzModRing(fmpz(2))`)
+# `FqDefaultPolyRing` (from `FqDefaultFiniteField(fmpz(2), 3, :x)`)
+
+function _iso_oscar_gap_polynomial_ring_functions(RO::PolyRing{T}, RG::GAP.GapObj, coeffs_iso::MapFromFunc) where T
+   fam = GAP.Globals.ElementsFamily(GAP.Globals.FamilyObj(codomain(coeffs_iso)))
+   ind = GAP.Globals.IndeterminateNumberOfUnivariateRationalFunction(
+           GAP.Globals.IndeterminatesOfPolynomialRing(RG)[1])
```

Comment:
Let's help Julia deduce the right types
```suggestion
   fam = GAP.Globals.ElementsFamily(GAP.Globals.FamilyObj(codomain(coeffs_iso)))::GapObj
   ind = GAP.Globals.IndeterminateNumberOfUnivariateRationalFunction(
           GAP.Globals.IndeterminatesOfPolynomialRing(RG)[1])::GapObj
```